src/server/services/wms/qgswmsgetprint.cpp

```diff
@@ -39,6 +39,10 @@ namespace QgsWms
     {
       contentType = "image/png";
     }
+    else if ( format == QgsWmsParameters::JPG )
```

Comment:
This should really be a switch to get warnings and avoid bugs like this

Reply:
Changes done